magento/magento2-login-as-customer
# Refactor LoginAsCustomer Login Model

### Description (*)
Model Magento\LoginAsCustomer\Model\Login has a lot of methods, were needed to move all that logic to separate PHP classes.

### Related Pull Requests
https://github.com/magento/magento2-login-as-customer/pull/91


### Manual testing scenarios (*)
All should work as worked before, except Login As Customer Log as now the secret key record is removed after successful login. Log logic will be implemented in separate PR.

### Fixed Issues (if relevant)
https://github.com/magento/magento2-login-as-customer/issues/97 Refactor Magento\LoginAsCustomer\Model\Login Model


### Contribution checklist (*)
 - [+] Pull request has a meaningful description of its purpose
 - [+] All commits are accompanied by meaningful commit messages
